app/modules/submissions/models.py

```diff
@@ -515,3 +516,10 @@ def delete(self):
         db.session.refresh(self)
         with db.session.begin():
             db.session.delete(self)
+
+    @classmethod
+    def tus_upload_dir(cls, guid=None):
+        d = os.path.join('_db', 'uploads')
```

Comment:
We need a new TUS config class defined in `config.py` for things like this to specify the upload path on disk

Reply:
@bluemellophone  -  just fyi, i have made this change as part of my current feature branch for sightings.  look for it in a develop branch near you soon!   thanks for the suggestion

https://github.com/WildMeOrg/houston/blob/sighting_passthrough_basic/app/extensions/tus/__init__.py#L64